src/routes/accommodation.route.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import authMiddleware from '../middlewares/auth.middleware';
+import AccommodationController from '../controllers/accommodation.controller';
+import bookAccommodationValidation from '../validations/accommodation.validation';
+
+const router = express.Router();
+
+router.post('/book-accommodation', authMiddleware.checkUserLoggedIn, bookAccommodationValidation, AccommodationController.createAccommodation);
```

Comment:
This route should be `POST /accommodations` instead. Refer to the REST best practices link I shared some weeks ago.